sdc_lifecycle.html

```diff
@@ -0,0 +1,192 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <!-- Font Awesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <!-- USWDS Styles -->
+  <link rel="stylesheet" href="assets/css/uswds.min.css">
+  <!-- SDC Styles -->
+  <link rel="stylesheet" href="css/styles.css">
+  <!-- Redesign Styles -->
+  <link rel="stylesheet" href="css/redesign_styles.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script>
+    $(document).ready(function(){
+      $('#footer').load('footer.html');
+      $('#banner').load('banner.html');
+      $('#header').load('header.html');
+    });
+  </script>
+  <div id="header"></div>
+</head>
+
+<body>
+<div id="banner" data-current-page="about_menu"></div>
+<main >
+  <section>
+    <div class="grid-container">
+      <div class="grid-row grid-gap margin-y-5">
+        <div class="tablet:grid-col-2">
+          <nav aria-label="Secondary navigation">
+            <ul class="usa-sidenav">
+              <li class="usa-sidenav__item">
+                <a href="about.html">What is SDC?</a>
+              </li>
+              <li class="usa-sidenav__item">
+                <a href="" class="usa-current">SDC Lifecycle</a>
+              </li>
+              <li class="usa-sidenav__item">
+                <a href="current_data.html">Current Data Available</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="tablet:grid-col-10">
+          <h1 class="margin-top-0 margin-bottom-1" style="font-weight: normal;">SDC Lifecycle</h1>
+          <p class="text-normal">Each project in the SDC goes through a standard lifecycle of activities.</p>
+        </div>
+        <div class="lifecycleInfoCard tablet:grid-offset-2 tablet:grid-col-8 margin-bottom-neg-2" >
+          <div style="border: solid #9DDFEB 8px;" >
```

Comment:
Can you remove all inline styles? This `9DDFEB` color in particular is in a couple places...